src/auth/oauth2/qgsauthoauth2edit.cpp

```diff
@@ -633,11 +633,11 @@ void QgsAuthOAuth2Edit::removeTokenCacheFile()
     return;
   }
 
-  QStringList cachefiles;
-  cachefiles << QgsAuthOAuth2Config::tokenCachePath( authcfg, false )
-             << QgsAuthOAuth2Config::tokenCachePath( authcfg, true );
+  QStringList cachefiles = QStringList()
```

Comment:
oauth2 seems to be maintained 
see https://github.com/golang/oauth2
what is the strategy here?

It's a bit early to talk about git submodules again, but I fail to understand to have hard unmaintained copies of a maintained library. Or am I missing something?

It's a not strictly related to this PR and am not against merging it.

Reply:
Isn't the part you are talking about in the external o2 folder and not in here?

Reply:
oops, indeed. sorry for the noise.